packages/components/tests/integration/components/hds/dropdown/list-item/checkbox-test.js

```diff
@@ -0,0 +1,66 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module(
+  'Integration | Component | hds/dropdown/list-item/checkbox',
```

Comment:
I see below we are testing the "selected" state of the "checkmark" which made me think: should we test also the "checked" state of the `list-item/checkbox`? (and radio too?)

Reply:
also, should we test the `getId` functionality too? (the label and control having the correct ID?)

Reply:
amended the tests in [121b639](https://github.com/hashicorp/design-system/pull/1185/commits/121b6396c33a83bca71f09c1805ad12f7de8fb43) to capture these features